packages/hoppscotch-app/components/realtime/Mqtt.vue

```diff
@@ -32,101 +32,331 @@
+      >
+        <img
+          :src="`/images/states/${$colorMode.value}/login.svg`"
+          loading="lazy"
+          class="inline-flex flex-col object-contain object-center w-16 h-16 my-4"
+          :alt="`${t('empty.authorization')}`"
+        />
+        <span class="pb-4 text-center">
+          {{ t("mqtt.connection_not_authorized") }}
+        </span>
+        <ButtonSecondary
+          outline
+          :label="t('app.documentation')"
+          to="https://docs.hoppscotch.io/features/authorization"
+          blank
+          svg="external-link"
+          reverse
+          class="mb-4"
         />
       </div>
-      <div class="flex items-center justify-between p-4">
-        <label for="mqtt-message" class="font-semibold text-secondaryLight">
-          {{ $t("mqtt.communication") }}
-        </label>
+
+      <div v-else class="flex flex-1 border-b border-dividerLight">
+        <div class="w-2/3 border-r border-dividerLight">
+          <div v-if="authType === 'basic'">
+            <div class="flex flex-1 border-b border-dividerLight">
+              <SmartEnvInput
+                v-model="username"
+                :placeholder="t('authorization.username')"
+              />
+            </div>
+            <div class="flex flex-1 border-b border-dividerLight">
+              <SmartEnvInput
+                v-model="password"
+                :placeholder="t('authorization.password')"
+              />
+            </div>
+          </div>
+          <div v-if="authType === 'token'">
+            <div class="flex flex-1 border-b border-dividerLight">
+              <SmartEnvInput
+                v-model="token"
+                :placeholder="t('authorization.token')"
+              />
+            </div>
+          </div>
+        </div>
+        <div
+          class="sticky h-full p-4 overflow-auto bg-primary top-upperTertiaryStickyFold min-w-46 max-w-1/3 z-9"
+        >
+          <div class="p-2">
+            <div class="pb-2 text-secondaryLight">
+              {{ t("helpers.authorization") }}
+            </div>
+            <SmartAnchor
+              class="link"
+              :label="`${t('authorization.learn')} \xA0 →`"
+              to="https://docs.hoppscotch.io/features/authorization"
+              blank
+            />
+          </div>
+        </div>
       </div>
-      <div class="flex px-4 space-x-2">
-        <input
-          id="mqtt-message"
-          v-model="message"
-          class="input"
-          type="text"
-          autocomplete="off"
-          :placeholder="$t('mqtt.message')"
-          spellcheck="false"
-        />
-        <ButtonPrimary
-          id="publish"
-          name="get"
-          :disabled="!canPublish"
-          :label="$t('mqtt.publish')"
-          @click.native="publish"
-        />
+    </template>
+    <template #secondary>
+      <div class="h-full flex flex-col hide-scrollbar !overflow-auto">
+        <div
```

Comment:
These tabs stuff should be in a separate component as well.
If the MQTT component is to deal with the rendering the whole of the MQTT UI, Tab logic should not be its concern!!

Single Response Principle please!!!!!